softdev.bib

```diff
@@ -3335,19 +3335,36 @@ @article{yanovski21ghostcell
 }
 
 @inproceedings{cheng20typefreezing,
-    title = "Type freezing: exploiting attribute type monomorphism in tracing {JIT} compilers",
-    author = "Cheng, Lin and Ilbeyi, Berkin and Bolz-Tereick, Carl Friedrich and Batten, Christopher",
-    year = 2020,
-    isbn = {9781450370479},
-    month = Feb,
-    booktitle = "CGO"
+  title = "Type freezing: exploiting attribute type monomorphism in tracing {JIT} compilers",
+  author = "Cheng, Lin and Ilbeyi, Berkin and Bolz-Tereick, Carl Friedrich and Batten, Christopher",
+  year = 2020,
+  isbn = {9781450370479},
+  month = Feb,
+  booktitle = "CGO"
 }
 
 @inproceedings{brunthaler10quickening,
-    title = "Efficient Interpretation using Quickening",
-    author = "Brunthaler, Stefan",
-    year = 2010,
-    month = Oct,
-    doi={10.1145/1869631.1869633},
-    booktitle = {DLS}
+  title = "Efficient Interpretation using Quickening",
+  author = "Brunthaler, Stefan",
+  year = 2010,
+  month = Oct,
+  doi = {10.1145/1869631.1869633},
+  booktitle = {DLS}
+}
+
+@inproceedings{bolz07not,
```

Comment:
could not find `booktitle` for this one...

Reply:
"3rd Workshop on Dynamic Languages and Applications" according to https://scholar.google.com/citations?view_op=view_citation&hl=en&user=S0rpYpkAAAAJ&citation_for_view=S0rpYpkAAAAJ:8k81kl-MbHgC

Reply:
Ah, I thought we keep `booktitle` for events/conferences. 
Are journals basically the same thing?

Reply:
From memory, we use 'booktitle' to mean "event" (you can look elsewhere in the file for precedent).

Reply:
Updated :point_right: 6770bc91682d12daabe142fb83793807ea26c05d